### DISCUSSION TITLE
Adding Azure Monitor Support to otel

### DISCUSSION THREAD

**Discussion by litan1106** [roadrunner-server/roadrunner#1608] [Ideas]
Hi,

Please add Azure Monitor Support so we can use roadrunner with Azure Application Insight with otel

https://github.com/roadrunner-server/otel
https://pkg.go.dev/github.com/open-telemetry/opentelemetry-collector-contrib/exporter/azuremonitorexporter

**Comment by rustatian** [comment]
Hey @litan1106 👋🏻 
Good idea, feel free to create a FR 👍🏻
